devel/ci/integration/tests/utils.py

```diff
@@ -61,3 +61,17 @@ def edit_file(container, path):
         container.copy_from(path, destfile)
         yield destfile
         container.copy_to(destfile, path)
+
+
+def build_container_image(docker_backend, app_name):
+    """Build the container image if necessary."""
+    container_name = "bodhi-ci-{}".format(app_name)
+    if container_name not in [c.name for c in docker_backend.list_images()]:
+        print("Building container image for {}".format(app_name))
+        subprocess.check_call([
+            "docker", "build", "--pull", "-t", container_name,
+            "-f", "devel/ci/integration/{}/Dockerfile".format(app_name), ".",
+        ])
```

Comment:
Why not have ```bodhi-ci``` do this, so it can be done in parallel with all the other build tasks?